test/RelayPricer.test.ts

```diff
@@ -0,0 +1,104 @@
+import chai, { assert, expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { StubbedInstance, stubInterface } from 'ts-sinon';
+
+import { RelayPricer } from '../src';
+import CoinBase from '../src/api/CoinBase';
+import { SourceApi } from '../src/types/RelayPricer';
+
+chai.use(chaiAsPromised);
+
+describe('RelayPricer', () => {
+    let fakeSourceApi: StubbedInstance<SourceApi>;
+    let pricer: RelayPricer;
+    const FAKE_PRICE_1 = 23345.834630269488;
+    const FAKE_PRICE_2 = 0.0775886924643371;
+    const FAKE_PRICE_CONVERSION_1 = 300892.2291221775;
```

Comment:
ah, I see what happened.
Perhaps, don't call it price conversion, I think that's misleading. Instead I'd call it exchange rate (or `xRate` for short) Also define what rate that is, eg `xRateUsdRbtc`.
also fake price name is confusing. This is a test. all data is fake, so no need to call it that.

Reply:
also don't just make numbers of which calculations you're testing up. that can lead to false positives more easily.
if you want to test different values, have different made up currencies and calculate their ratio.

Reply:
one last thing here, I'd say that only constants defined outside of all scope (within a module, ie file) should be named with all caps. Meaning, that constants defined inside a describe block should follow normal lower camel casing.

Reply:
updated